### PR TITLE
[RUNTIME][CLML] Fix for CLML ops and enable more test case

### PR DESCRIPTION
**Pull request by krishnaraj36**
CLML multiple bug fix and restructuring
1. Fixed the dense op failure in network test case.
2. Fixed the bugs for Float16 test case
3. Restructure the test cases for CLML with more coverage.

### CODE REVIEW

**Comment by krishnaraj36** [comment]
@srkreddy1238 : Please review this PR

**Comment by krishnaraj36** [comment]
@srkreddy1238 : please review this PR

**Comment by krishnaraj36** [comment]
@srkreddy1238 @echuraev : Can you please take a look this PR and let me know your feedback.

File: tests/scripts/task_python_adreno.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ export TVM_TRACKER_PORT=$(((RANDOM % 100) + 9100))
 export RPC_DEVICE_KEY="android"
 export RPC_TARGET="adreno"
 export TVM_NDK_CC="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android28-clang"
+export CXX="${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android28-clang"
```
Comment (srkreddy1238) [inline tests/scripts/task_python_adreno.sh:34]: CXX change may create issue for cython3 build.

**Review by srkreddy1238 — CHANGES_REQUESTED** [review]
Overall looks good.
Let the CI pass.

File: tests/scripts/task_python_adreno.sh (hunk 1 above)
Comment (krishnaraj36) [reply tests/scripts/task_python_adreno.sh:34]: > CXX change may create issue for cython3 build.

Can you specify which build can fail; CI got pass for this change.

**Comment by krishnaraj36** [comment]
@echuraev  Can you please take a look this PR and let me know your feedback

**Review by srkreddy1238 — APPROVED** [review]
LGTM

**Pull request merged**